natgeo/mortar
# [SYTO-753+SYTO-772] Style POD component and animate hover

https://jira.natgeo.com/browse/SYTO-753
https://jira.natgeo.com/browse/SYTO-772

This story styles the POD component created in SYTO-771.

To test:

- Pull SYTO-753
- Verify unit tests pass by running `npm test`
- Run `gulp serve`
- Verify component at http://localhost:3000/components/podpromo/index.html using the configurator
- Verify component also renders as expected at http://localhost:3000/components/
